src/arkit.cs

```diff
@@ -1395,6 +1584,373 @@ interface ARWorldMap : NSCopying, NSSecureCoding {
 		[Export ("rawFeaturePoints", ArgumentSemantic.Strong)]
 		ARPointCloud RawFeaturePoints { get; }
 	}
+
+	[iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARBody2D {
+
+		[Export ("skeleton")]
+		ARSkeleton2D Skeleton { get; }
+	}
+
+	[iOS (13,0)]
+	[BaseType (typeof(ARAnchor))]
+	[DisableDefaultCtor]
+	interface ARBodyAnchor : ARTrackable {
+
+		[Export ("initWithAnchor:")]
+		IntPtr Constructor (ARAnchor anchor);
+
+		// [Export ("initWithTransform:")] marked as NS_UNAVAILABLE
+		// [Export ("initWithName:")] marked as NS_UNAVAILABLE
```

Comment:
Do we want this comment here or in the xtro file?

Reply:
there's no entry to comment and while comments alone can be added it will never be updated (even if marked available later) since it won't be reported